apps/src/code-studio/pd/workshop_dashboard/local_summer_workshop_survey_results.jsx

```diff
@@ -110,21 +161,48 @@ const LocalSummerWorkshopSurveyResults = React.createClass({
     }
   },
 
+  renderFreeResponseBullets(question, answerCollection) {
+    if (question['facilitator_breakdown'] && this.state.facilitatorBreakdown) {
+      return this.freeResponseMapToBullets(answerCollection);
+    } else {
+      return answerCollection.map((answer, i) => {
+        return (
+          <li key={i}>
+            {answer}
+          </li>
+        );
+      });
+    }
+  },
+
+  freeResponseMapToBullets(answer) {
+    return Object.keys(answer).map((facilitator_name, i) => {
+      return (
+        <li key={i}>
+          {facilitator_name}
+          <ul>
+            {
+              answer[facilitator_name].map((feedback, j) => {
+                return (
+                  <li key={j}>
+                    {feedback}
+                  </li>
+                );
+              })
+            }
+          </ul>
+        </li>
+      );
+    });
+  },
+
   renderFreeResponseFeedback() {
     const freeResponseAnswers = freeResponseQuestions.map((question, i) => {
-      return (
+      let answerCollection = this.state.thisWorkshop[question['key']];
+      return answerCollection && (
         <div key={i} className="well">
```

Comment:
Why don't we use the [Well](https://react-bootstrap.github.io/components.html#wells) `React-Bootstrap` class? 

Incidentally React-Bootstrap does rely on external styles, but we use it throughout the workshop dashboard and those style references are internal to React-Bootstrap. We don't have to specify class names in our code.